### PR TITLE
Add introduction and links to the new guide to the vLLM optimized Doc…

### PR DESCRIPTION
**Pull request by amd-jnovotny**
…ker image on AMD Infinity Hub

Three guides were updated with a link to the new guide (URL not yet known) and a very brief introduction.

This guide is currently linking back to the main AMD Infinity Hub because we don't know the actual link yet. This must be updated later.

### CODE REVIEW

**Review by cindylee2 — COMMENTED** [review]
The changes look good. Once we have the Infinity hub URL, I will share it with you.

**Comment by peterjunpark** [comment]
> The changes look good. Once we have the Infinity hub URL, I will share it with you.

Hi @cindylee2, do you have the specific Infinity Hub URL?

**Comment by cindylee2** [comment]
Please use this link instead of the Infinity Hub URL. We'll directly send users to the GitHub page. 
https://github.com/ROCm/MAD/vllm_benchmark.md

File: docs/how-to/llm-fine-tuning-optimization/llm-inference-frameworks.rst [hunk 1]
```diff
@@ -137,6 +137,12 @@ Installing vLLM
 
 Refer to :ref:`mi300x-vllm-optimization` for performance optimization tips.
 
+ROCm provides a prebuilt optimized Docker image for validating the performance of LLM inference with vLLM 
+on the MI300X accelerator. The Docker image includes ROCm, vLLM, PyTorch, and tuning files in the CSV 
+format. For more information, see the guide to 
+`LLM inference performance validation with vLLM on the AMD Instinct™ MI300X accelerator <https://github.com/ROCm/MAD/vllm_benchmark.md>`_
```
Comment (seungrokj) [inline docs/how-to/llm-fine-tuning-optimization/llm-inference-frameworks.rst:143]: @cindylee2 FYI too 
https://github.com/ROCm/MAD/benchmark/vllm/README.md

**Comment by seungrokj** [comment]
> Please use this link instead of the Infinity Hub URL. We'll directly send users to the GitHub page. https://github.com/ROCm/MAD/vllm_benchmark.md

Hi @cindylee2  the link has changed to 
https://github.com/ROCm/MAD/benchmark/vllm/README.md
FYI.

**Review by amitkumar-amd — APPROVED** [review]

**Pull request merged**